lib/widgets/message_list.dart

```diff
@@ -1262,15 +1270,24 @@ class MessageWithPossibleSender extends StatelessWidget {
           if (senderRow != null)
             Padding(padding: const EdgeInsets.fromLTRB(16, 2, 16, 0),
               child: senderRow),
-          EditStateMarker(
-            editState: message.editState,
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.baseline,
+            textBaseline: localizedTextBaseline(context),
             children: [
+              const SizedBox(width: 16),
               Expanded(child: Column(
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: [
                   MessageContent(message: message, content: item.content),
                   if ((message.reactions?.total ?? 0) > 0)
-                    ReactionChipsList(messageId: message.id, reactions: message.reactions!)
+                    ReactionChipsList(messageId: message.id, reactions: message.reactions!),
+                  if (editStateText != null)
+                    Text(editStateText, textAlign: TextAlign.end,
+                      style: TextStyle(
+                        color: messageListTheme.editStateLabel,
+                        fontFamily: 'Source Sans 3',
```

Comment:
I think we'll get 'Source Sans 3' automatically here, like in most other places, because this is rendered on a `Material` (via `Scaffold`, I think), and `Material` [applies `TextTheme.bodyMedium`](https://github.com/flutter/flutter/blob/a09941dcf1bae5196c8107dfced5d9c02c121905/packages/flutter/lib/src/material/material.dart#L467) to its children via `AnimatedDefaultTextStyle`, and we've set `TextTheme.bodyMedium` to include 'Source Sans 3' (see `zulipTypography` in lib/widgets/text.dart).

Reply:
Looks like we're also redundantly passing 'Source Sans 3' where we render the sender name in `MessageWithPossibleSender`. Removing that (perhaps with a link to my explanation here, or quoting/paraphrasing it) would be a helpful cleanup in a separate NFC commit.